03week/planet/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<header>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <!-- This CSS-reset link is canceled out for this project<link rel="stylesheet" href="css/css-reset.css" />-->
+    <link rel="stylesheet" href="css/style.css" />
+
+    <h1>The Daily Planet</h1>
+    <!--Can use nav-> ul -> li for a nav bar but makes up and down. Span goes left to right-->
+    <nav>
+        <span>
+    <span><a href="sports.asp">Sports</a></span>
+        <span><a href="local.asp">Local</a></span>
+        <span><a href="economy.asp">Economy</a></span>
+        </span>
+    </nav>
+
+</header>
+<body>
+    <main>
+        <section>
+          <article>
+            <!-- Resize the image to fit in-->
+            <div id="cage"><img src="img/cage-satan.jpg"></div>
```

Comment:
Hey bro! Try putting the id cage inside your img tag something like this <img src="img/cage-satan.jpg" id="cage"> and you should be be able to resize it. I hope that helps!